### DISCUSSION TITLE
how to pass on authentication info with langflow

### DISCUSSION THREAD

**Discussion by phoenix-lee-tpe** [langflow-ai/langflow#1195] [Q&A]
how do i pass authentication information, such as cookies, in a chatflow created by langflow, so i can do access control?

**Comment by Cristhianzl** [comment]
Hi! 👋

We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.

[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)

Thank you for your understanding.

Best regards,
The Langflow Team
